src/styles/filelist.styl

```diff
@@ -211,6 +211,10 @@ column-width-thumbnail-bigger = 7rem
         max-width 6rem
         margin 0 auto
 
+    .fil-file-description
+        max-width 6rem
+        overflow hidden
+
```

Comment:
I don't know, but we had in the past, a component for that: https://docs.cozy.io/cozy-ui/react/#/Legacy?id=filepath 

But i don't know why this component is in Legacy now